### DISCUSSION TITLE
Nuxt 3 Integration

### DISCUSSION THREAD

**Discussion by kelvinmatheus** [nuxt/nuxt#18054] [Questions] [answered]
We are planning to create a new `Vue 3` project using `Nuxt 3` with `Vite` as a bundler.

Do you guys have any documentation on how to use both together? I have seen the installation documentation to both of the to Nuxt 2, but I did not find to `Nuxt 3`. It says in the documentation that it has integration with `Vite`. So, I was wondering how to install it.

We are going to use with `Nuxt 3`:

 Vue 3 +  Vuex 4 + Vue Router 4 + TypeScript

**Comment by yassilah** [comment, marked as answer]
Nuxt 3 uses Vite by default, you don't need to add anything. As for the configuration, it's in the docs: https://v3.nuxtjs.org/docs/directory-structure/nuxt.config#vite 🙂

**Reply by itshizhan** [reply]
Vuex 4  is not supported by now, you can use pinia: https://github.com/vuejs/pinia

**Reply by kelvinmatheus** [reply]
> Vuex 4 is not supported by now, you can use pinia: https://github.com/vuejs/pinia

Nuxt 3 does not support Vuex 4?

**Reply by danielroe** [reply]
You could use vuex 4 but it isn't integrated into Nuxt. Pinia is recommended.

**Reply by kelvinmatheus** [reply]
> You could use vuex 4 but it isn't integrated into Nuxt. Pinia is recommended.

Thanks for the info. The team decided to stick with Vuex 4 for now as it is what we have most experience with. We are planning to migrate to `pinia` in case we se the Vue community moving to it, and we have some courses about it on Vue Mastery/Vue School.

Although Nuxt 3 is not production ready, we plan to use it in a hybrid app where some admin pages are develop with Nuxt 3. Those pages would be used by our company internal members. 

Although in the documentation does not mentioned on the `Deployment` section AWS as an option, we will be deploying to it. :)

Eventually, making a hybrid mobile and web app with integration with `Vuetify`, `Ionic Vue/Capacitor` if that is possible.

**Reply by danielroe** [reply]
That sounds pretty cool. Keep us up-to-date!
